tests/com_wrapper_int/test_object.c

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#include <stddef.h>
+#include <stdlib.h>
+
+#include "c_logging/logger.h"
+
+#include "c_pal/gballoc_hl.h"
+#include "c_pal/gballoc_hl_redirect.h"
+
+#include "test_object.h"
+
+
+typedef struct TEST_OBJECT_TAG
+{
+    char* a;
+} TEST_OBJECT;
+
+TEST_OBJECT_HANDLE test_object_create(const char* a)
+{
+    TEST_OBJECT_HANDLE result = malloc(sizeof(TEST_OBJECT));
+    if (result == NULL)
+    {
+        LogError("malloc failed");
+    }
+    else
+    {
+        size_t length = strlen(a) + 1;
+        result->a = malloc(length);
+        if (result->a == NULL)
```

Comment:
>if (result->a == NULL) [](http://example.com/codeflow?start=8&length=22)

Can we change this to be ASSERT_IS_NOT_NULL.  That way we would catch failures and not have to hunt.